lib/rex/exploitation/cmdstager/jsp.rb

```diff
@@ -0,0 +1,120 @@
+# -*- coding: binary -*-
+
+class Rex::Exploitation::CmdStagerJSP < Rex::Exploitation::CmdStagerBase
+  JAVA_KEYWORDS = %w[abstract assert boolean break byte case catch char class const continue default
+                     do double else enum extends final finally float for goto if implements import
+                     instanceof int interface long native new package private protected public return
+                     short static strictfp super switch synchronized this throw throws transient try
+                     void volatile while true false null var _
+                     out page pageContext config request response session application] # JSP-specific
```

Comment:
Just spotting this PR; Looks like we might want to look into a solution for this PR and for https://github.com/rapid7/metasploit-framework/pull/15436

Potentially aligning with the solution suggested by Spencer https://github.com/rapid7/metasploit-framework/pull/15436#issuecomment-881451637

Reply:
Nice suggestion, didn't knew `rex-random_identifier` existed, updated

Reply:
I'll add a commit as soon as that PR gets merged.

Reply:
It's merged now :+1: